### ISSUE TITLE
Can't Change the Emoji for the Horizontal Line

### ISSUE THREAD

**Issue by Addictiffane** [SlRvb/Obsidian--ITS-Theme#350]
I'm trying to change the emoji for the horizontal line, but I can't figure it out. I tried changing the settings but I can't figure out what to put. I'm not even sure if it's working because it shows in the settings that a lightning bolt should be there, yet I have an hourglass.

**Comment by marc-signer** [comment]
I second that. For me the horizontal line does not show an icon at all, even when trying to change it. Trying to change the color does not do anything for me either, even when digging into the theme's .css directly I'm not able to do it. I'm not sure if I've just missed something but the line really does look vastly different (red, no icon, fade-out on edges) than what it's supposed to.

[**This reddit post**](https://www.reddit.com/r/ObsidianMD/comments/1dppe7v/its_theme_horizontal_lines/) explains the problem I'm facing, which seems to be connected to yours, if I had to guess.

**Comment by SlRvb** [comment]
Hmm, I can't seem to recreate this. You're not removing the quotes in the style settings right?

**Comment by Addictiffane** [comment]
> Hmm, I can't seem to recreate this. You're not removing the quotes in the style settings right?

For me, I found the main issue. When you apply a certain alternate color scheme in style settings, it forces that themes icon. If I change the color scheme to default, it works just fine. 

Also, I don't understand how to find the codepoint for specific icons. And I was wondering if there was any way I could add a custom icon.

**Comment by marc-signer** [comment]
I played around with the Style Settings Plugin some more and realized that the "No Horizontal Line Symbol" button actually hides my entire horizontal line. Might be related in some way.

**Comment by Addictiffane** [comment]
> I played around with the Style Settings Plugin some more and realized that the "No Horizontal Line Symbol" button actually hides my entire horizontal line. Might be related in some way.

If you're using the WOTC theme, I believe the icon is a red bar.

**Comment by Addictiffane** [comment]
I found the codepoints of the icons for RPG-Awsome, but they don't seem to work.

**Comment by marc-signer** [comment]
> If you're using the WOTC theme, I believe the icon is a red bar.

You're right! Thank you so much. That's the pointer I needed and obviously a major oversight on my end, sorry for that - the icon is now showing, and the the recoloring works as well if I switch out the WOTC theme.
I feel like the horizontal line in the WOTC theme should be re-colorable in some form though, as of right now it does not seem to work.

> I found the codepoints of the icons for RPG-Awsome, but they don't seem to work.

Unless I'm completely misunderstanding something, I think only code points for the supported icon packs work (as in the dropdown in Style Settings just below the Icon). RPG-Awesome would definitely be cool though!
